tests/backend/realm/test_block.py

```diff
@@ -356,13 +368,13 @@ async def test_block_create_bad_msg(alice_ws, bad_msg):
     await alice_ws.send(packb({"cmd": "block_create", **bad_msg}))
     raw_rep = await alice_ws.receive()
     rep = block_create_serializer.rep_loads(raw_rep)
-    assert rep["status"] == "bad_message"
+    assert isinstance(rep, BlockCreateRepUnknownStatus)
```

Comment:
this check is not equivalent to the previous one, we must check the status is `bad_message`

so I guess we shoul do something like `rep == BlockCreateRepUnknownStatus("bad_message")`

Reply:
We use `assert rep.status == "bad_message"` instead, because the only `Rep` which have the field `status` are `UnknownStatusRep `